### PR TITLE
Fix devise password change error #165028596

### PR DESCRIPTION
**Pull request by software-project**
https://www.pivotaltracker.com/story/show/165028596

### CODE REVIEW

File: spec/controllers/overrides/passwords_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    end
+  end
+
+  describe '#edit' do
+    let(:token) do
+      allow_any_instance_of(User).to receive(:send_devise_notification).and_return(true)
+      @token ||= user.send_reset_password_instructions
+    end
+
+    it 'change allow_password_change to true' do
```
Comment (akegan) [inline spec/controllers/overrides/passwords_controller_spec.rb]: i'm a little confused about the logic around allow_password_change, why do we set it to true on edit?

**Comment by akegan** [comment]
are there any steps we need to do on deploy to handle the DB update?

File: spec/controllers/overrides/passwords_controller_spec.rb (hunk 1 above)
Comment (software-project) [reply spec/controllers/overrides/passwords_controller_spec.rb]: This is devise behavior. We are expecting it to be set to true. I presume it has something to do with safety. This functionality is available for unsigned users, so without token unuathorized user is not able to change password for some other account.

**Comment by software-project** [comment]
@akegan just run the db migration. I believe heroku doest it for us.

**Comment by akegan** [comment]
got QA sign-off from emily: merging and deploying tonight

**Pull request merged**